psa_builder.py

```diff
@@ -79,7 +74,7 @@
 ROOT = abspath(dirname(__file__))
 mbed_path = join(ROOT, "mbed-os")
 TF_M_RELATIVE_PATH = (
-    "platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_V1_1"
+    "platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST"
```

Comment:
Why is this LATEST and not 1_2?

Reply:
This was changed as part of the review process of https://github.com/ARMmbed/mbed-os/pull/14187, it made sense to keep the only version which is latest.

Reply:
@urutva's suggested that we avoid having to rename the whole directory for every TF-M update

Reply:
That makes sense